checker/checker.go

```diff
@@ -261,6 +264,21 @@ func (c *Checker) Process(ctx context.Context, pr chan pb.ProcessResult) {
 	}
 }
 
+// updateInstruction updates the check result's Instruction
+func (c *Checker) updateInstruction(result *check.Results) {
+	for _, r := range result.Results {
+		if r.State == check.StateSuccess {
+			continue
+		}
+
+		// can't judge by other field, maybe update it later
+		switch r.Extra {
+		case check.AutoIncrementKeyChecking:
+			r.Instruction = "please handle it by yourself, read document https://pingcap.com/docs-cn/dev/reference/tools/data-migration/usage-scenarios/best-practice-dm-shard/#自增主键冲突处理 for more detail (only have Chinese document now, will translate to English later)"
```

Comment:
will you merge this PR after the doc translated to English or ASAP?

Reply:
I think it is better merge ASAP, don't know when l18n finish translate.